opentelemetry-sdk/src/opentelemetry/sdk/resources/__init__.py

```diff
@@ -146,8 +146,6 @@ def __init__(
         self, attributes: Attributes, schema_url: typing.Optional[str] = None
     ):
         self._attributes = BoundedAttributes(attributes=attributes)
-        if schema_url is None:
-            schema_url = ""
```

Comment:
This was a requirement from spec https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/sdk.md#create?

Reply:
Well, it says empty if unspecified (not sure if that specifically means `""`). I prefer to use `None` to indicate that `schema_url` is unspecified because having `""` could mean `schema_url` was not specified or it was specifically set to `""`.

Reply:
> it says empty if unspecified (not sure if that specifically means "")

I think it means an empty string here otherwise it would have explicitly mentioned it should be set to null.

Reply:
Ok, changed the default value to `""` everywhere. [Go](https://github.com/open-telemetry/opentelemetry-go/blob/main/sdk/resource/resource.go#L133) seems to do the same.